.circleci/config.yml

```diff
@@ -315,3 +393,9 @@ workflows:
             branches:
               only:
                 - master
+      - Deploy to Rinkeby with Gnosis and Vault:
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
Since `v2.0.0` is the default branch, should we check deployments in the branch too

Reply:
uh uh, forgot about that!